.github/workflows/regression.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         config_set: [BaseMPI, ReverseMPI, ForwardMPI, BaseNoMPI, ReverseNoMPI, ForwardNoMPI, BaseOMP, ReverseOMP, ForwardOMP]
         include:
           - config_set: BaseMPI
-            flags: '-Denable-pywrapper=true -Denable-tests=true --warnlevel=3 --werror'
+            flags: '-Denable-pywrapper=true -Denable-coolprop=true -Denable-tests=true --warnlevel=3 --werror'
```

Comment:
Ok, in this file, please try replacing `docker://ghcr.io/su2code/su2/build-su2:220614-1237` with `ghcr.io/su2code/su2/build-su2:221027-0442`, cmake should be available.

Reply:
Thank you, I will try now

Reply:
There should be a `ninja` executable in the root dir of SU2, try pointing CoolProp to use it.
```
Configuring the build directory with CMake version 3.16.3
| Running CMake with: -G Ninja -DCMAKE_INSTALL_PREFIX=/github/workspace/install -DCOOLPROP_STATIC_LIBRARY=True
| - build directory:          /github/workspace/src/SU2_refs_pull_1782_merge/docker_build/subprojects/CoolProp/__CMake_build
| - source directory:         /github/workspace/src/SU2_refs_pull_1782_merge/subprojects/CoolProp
| - toolchain file:           /github/workspace/src/SU2_refs_pull_1782_merge/docker_build/subprojects/CoolProp/__CMake_build/CMakeMesonToolchainFile.cmake
| - preload file:             /github/workspace/src/SU2_refs_pull_1782_merge/docker_build/meson-private/data/preload.cmake
| - trace args:               --trace --trace-expand --no-warn-unused-cli --trace-redirect=cmake_trace.txt
| - disabled policy warnings: [CMP0025, CMP0047, CMP0056, CMP0060, CMP0065, CMP0066, CMP0067, CMP0082, CMP0089, CMP0102]

| Running with trace output on.
| Running with expanded trace output on.
| Not searching for unused variables given on the command line.
| Trace will be written to cmake_trace.txt
| -- COOLPROP_INSTALL_PREFIX=/github/workspace/src/SU2_refs_pull_1782_merge/subprojects/CoolProp/install_root
| CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
| -- Configuring incomplete, errors occurred!
| See also "/github/workspace/src/SU2_refs_pull_1782_merge/docker_build/subprojects/CoolProp/__CMake_build/CMakeFiles/CMakeOutput.log".
```

Reply:
I will try to solve this issue. 

Thank you for your advice.

Reply:
Ideally we would make everything work without extra dependencies for users.
I'll try to help.
I will move this branch to su2code:feature_coolprop, then please open a PR of that branch into develop (this way you remain the author and all your commit history is there, but it's easier for me to checkout the branch locally.